File: examples/availability/retrieve_hosted/retrieve_hosted.go
import (
	"os"

	"github.com/bloock/bloock-sdk-go/v2"
	"github.com/bloock/bloock-sdk-go/v2/client"
	"github.com/bloock/bloock-sdk-go/v2/entity"
)

func main() {
	bloock.ApiKey = os.Getenv("API_KEY")

	recordClient := client.NewRecordClient()
	availabilityClient := client.NewAvailabilityClient()

	id := "publish uuid result"
	record, err = recordClient.FromLoader(entity.NewHostedLoader(id)).Build()
	if err != nil {
		log.Prinln(err)
	}
}

File: examples/availability/retrieve_hosted/retrieve_hosted.java
import com.bloock.sdk.Bloock;
import com.bloock.sdk.client.AvailabilityClient;
import com.bloock.sdk.client.RecordClient;
import com.bloock.sdk.entity.HostedLoader;
import com.bloock.sdk.entity.Record;

class Test {
  public static void main(String[] args) throws Exception {
    RecordClient recordClient = new RecordClient();
    AvailabilityClient availabilityClient = new AvailabilityClient();

    String hash = "publish uuid result";
    Record record = recordClient.fromLoader(new HostedLoader(id)).build();
  }
}
